Files already downloaded and verified
Files already downloaded and verified
 2021-11-30 14:06:21,399 | INFO | root 	 classes_order
 2021-11-30 14:06:21,400 | INFO | root 	 [[87, 0, 52, 58, 44, 91, 68, 97, 51, 15, 94, 92, 10, 72, 49, 78, 61, 14, 8, 86, 84, 96, 18, 24, 32, 45, 88, 11, 4, 67, 69, 66, 77, 47, 79, 93, 29, 50, 57, 83, 17, 81, 41, 12, 37, 59, 25, 20, 80, 73, 1, 28, 6, 46, 62, 82, 53, 9, 31, 75, 38, 63, 33, 74, 27, 22, 36, 3, 16, 21, 60, 19, 70, 90, 89, 43, 5, 42, 65, 76, 40, 30, 23, 85, 2, 95, 56, 48, 71, 64, 98, 13, 99, 7, 34, 55, 54, 26, 35, 39]]
 2021-11-30 14:06:24,173 | INFO | root 	 Begin step 0
 2021-11-30 14:06:24,174 | INFO | root 	 Now [200, 200, 200, 200, 200, 200, 200, 200, 200, 200] examplars per class.
 2021-11-30 14:06:24,175 | INFO | root 	 Step 0 weight decay 0.00050
 2021-11-30 14:06:24,256 | INFO | root 	 Train on 0->10.
 2021-11-30 14:06:24,257 | INFO | root 	 nb 5000
 2021-11-30 14:06:24,258 | INFO | root 	 Initial trainset: Weight norm per class [1.007]
Traceback (most recent call last):
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/sacred/experiment.py", line 318, in run_commandline
    options=args,
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/sacred/experiment.py", line 276, in run
    run()
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/sacred/run.py", line 238, in __call__
    self.result = self.main_function(*args)
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/sacred/config/captured_function.py", line 42, in captured_function
    result = wrapped(*args, **kwargs)
  File "/home/share/jiawei/solf/codes/base/main.py", line 69, in train
    _train(cfg, _run, ex, tensorboard)
  File "/home/share/jiawei/solf/codes/base/main.py", line 120, in _train
    model.train_task(train_loader, val_loader)
  File "/home/share/jiawei/solf/inclearn/models/base.py", line 43, in train_task
    self._train_task(train_loader, val_loader)
  File "/home/share/jiawei/solf/inclearn/models/incmodel.py", line 252, in _train_task
    self._increments, "Initial trainset")
  File "/home/share/jiawei/solf/inclearn/tools/utils.py", line 87, in display_feature_norm
    avg_feat_norm_per_cls = get_featnorm_grouped_by_class(network, n_classes, loader)
  File "/home/share/jiawei/solf/inclearn/tools/utils.py", line 49, in get_featnorm_grouped_by_class
    for x, y in loader:
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 521, in __next__
    data = self._next_data()
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1203, in _next_data
    return self._process_data(data)
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 1229, in _process_data
    data.reraise()
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/_utils.py", line 425, in reraise
    raise self.exc_type(msg)
TypeError: Caught TypeError in DataLoader worker process 0.
Original Traceback (most recent call last):
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/_utils/worker.py", line 287, in _worker_loop
    data = fetcher.fetch(index)
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/_utils/fetch.py", line 47, in fetch
    return self.collate_fn(data)
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/_utils/collate.py", line 84, in default_collate
    return [default_collate(samples) for samples in transposed]
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/_utils/collate.py", line 84, in <listcomp>
    return [default_collate(samples) for samples in transposed]
  File "/home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/utils/data/_utils/collate.py", line 86, in default_collate
    raise TypeError(default_collate_err_msg_format.format(elem_type))
TypeError: default_collate: batch must contain tensors, numpy arrays, numbers, dicts or lists; found <class 'PIL.Image.Image'>
warmup
> /home/share/jiawei/anaconda3/envs/torch3090/lib/python3.7/site-packages/torch/_utils.py(425)reraise()
-> raise self.exc_type(msg)

(Pdb) [?2004l
